seastar/src/distributed.rs

```diff
@@ -304,6 +334,42 @@ impl<S: Service> Distributed<S> {
         })
     }
 
+    fn submit_to_mut<'a, Func, Fut, Ret>(
+        &'a self, // this immutable reference is intentional
```

Comment:
Why is it intentional? This comment piques reader's curiosity but doesn't satisfy it.

Reply:
True.